sys/include/net/netopt.h

```diff
@@ -340,6 +340,15 @@ typedef enum {
      */
     NETOPT_IQ_INVERT,
 
+    /**
+     * @brief   Get retry amount from missing ACKs of the last transmission
+     *
+     * This retrieves the number of retries needed for the last transmissions.
+     * Only retries due to missing ACK packets are considered. Retries due to
```

Comment:
to make this even more clear you could change this to
"Only **retransmissions** due to missing ACK packets are considered."